tests/test_price_parsing.py

```diff
@@ -1983,6 +2193,118 @@ def __eq__(self, other):
     #         '¥', '132', 132),
     Example('Free Shipping on Orders $49+.', 'Free Shipping on Orders $49+.',
             '$', None, None),
+
+]
+
+# Valid currencies not detected by price-parser
+# Move to regular tests when added
+PRICE_PARSING_EXAMPLES_XFAIL_CURRENCIES_TO_BE_ADDED = [
+    Example(None, '22000.00元/台',  # 元 is yuan
+            '元', '22000.00', 22000.00),
+    Example(None, '2963yen',
+            'yen', '2963', 2963),
+    Example(None, '1 บาท',  # Thai baht
+            'บาท', '1', 1.0),
+    Example(None, '1 ر.س',  # Saudi riyal
+            'ر.س', '1', 1.0),
+    Example(None, '1.198,- Kr',
+            'Kr', '1.198', 1198.0),
+    Example(None, '45 جنيه',  # Egyptian Pound (EGP)
+            'جنيه', '45', 45.0),
+    Example(None, '45 ج.م',  # Egyptian Pound (EGP)
+            'ج.م', '45', 45.0),
+    Example(None, '45 E£',  # Egyptian Pound (EGP)
+            'E£', '45', 45.0),
+    Example(None, '2000 zl',  # Polish zloty (PLN)
+            'zl', '2000', 2000.0),
+    Example(None, '14.00 SGD / Each',  # Singapore Dollar
+            'SGD', '14.00', 14.0),
+    Example(None, 'CAN$1.23',
+            'CAN$', '1.23', 1.23),
+    Example(None, '200q',  # Guatemalan Quetzal
+            'q', '200', 200.0),
+    Example(None, '200Q',  # Guatemalan Quetzal
+            'Q', '200', 200.0),
+    Example(None, '1 دج',  # Algerian Dinar
+            'دج', '1', 1.0),
+    Example(None, '1 .د.ب',  # Bahraini Dinar
+            '.د.ب', '1', 1.0),
+    Example(None, '1 $b',  # Barbadian dollar
+            '$b', '1', 1.0),
+    Example(None, '1 P',  # Pound sterling, Botswana pula
+            'P', '1', 1.0),
+    Example(None, '1 franc',
+            'franc', '1', 1.0),
+    Example(None, '2 francs',
+            'francs', '2', '2.0'),
+    Example(None, '1 ናቕፋ',  # Eritrean Nakfa
+            'ናቕፋ', '1', 1.0),
+    Example(None, '1 نافكا',  # Arabic "نافكا" stands for "Navka", which can refer to Eritrean Nakfa
+            'نافكا', '1', 1.0),
+    Example(None, '1 E',  # Can refer to Egyptian pound
+            'E', '1', 1.0),
+    Example(None, '1 Br',  # Ethiopian birr, Belarusian ruble
+            'Br', '1', 1.0),
+    Example(None, '1 ብር',  # Ethiopian birr
+            'ብር', '1', 1.0),
+    Example(None, '1 D',  # Gambian dalasi
+            'D', '1', 1.0),
+    Example(None, '1 GEL',  # Georgian lari
+            'GEL', '1', 1.0),
+    Example(None, '1 ლ',  # Georgian lari
+            'ლ', '1', 1.0),
+    Example(None, '1 ¢',  # Ghanaian cedi, cents
+            '¢', '1', 1.0),
+    Example(None, '1 ₵',  # Ghanaian cedi
+            '₵', '1', 1.0),
+    Example(None, '1 GFr',  # Guinean franc
+            'GFr', '1', 1.0),
+    Example(None, '1 FG',  # Guinean franc
+            'FG', '1', 1.0),
+    Example(None, '1 G',  # Actually "G" is too ambiguous and it does not clearly mean any currency
+            None, '1', 1.0),
```

Comment:
Why is it marked as xfail? It seems the current price-parser behavior is the expected one.

It seems there are some other XPASS tests:

```
FAILED tests/test_price_parsing.py::test_parsing[None, '14.00 SGD / Each']
FAILED tests/test_price_parsing.py::test_parsing[None, '1 Br'1]
FAILED tests/test_price_parsing.py::test_parsing[None, '1 GEL']
FAILED tests/test_price_parsing.py::test_parsing[None, '1 FG']
FAILED tests/test_price_parsing.py::test_parsing[None, '1 G']
FAILED tests/test_price_parsing.py::test_parsing[None, '1 R\u20a3']
``` 

(to see this, you need https://github.com/scrapinghub/price-parser/pull/54 - feel free to merge it if it looks ok to you)

Reply:
We should also fix CI (https://github.com/scrapinghub/price-parser/issues/52), it's not good that tests are not running automatically :(

Reply:
This was indeed needed for the cases I'm adding in this PR, because I'd like them to fail the test marked as xfail when they're fixed so we can easily keep track of them. Thank you!